package.json

```diff
@@ -147,7 +147,7 @@
     "JSONStream": "1.3.5",
     "abortcontroller-polyfill": "^1.4.0",
     "accept": "3.0.2",
-    "angular": "^1.7.9",
+    "angular": "^1.8.0",
     "angular-aria": "^1.7.9",
```

Comment:
@watson I just noticed this... I guess it's published from the same repo and also has a 1.8.0 update available. Do you want to bump that too while you're at it?

Reply:
Well spottet. I found a few others as well that I upgraded also